test/index.js

```diff
@@ -2,19 +2,16 @@
 
 var os = require('os');
 var path = require('path');
+var pipeline = require('stream').pipeline;
```

Comment:
`pipeline` actually rethrows errors in a pipeline (my PR 😛), so we might not need to separate the assert and the writeable stream. I haven't actually tested that yet.